drivers/clock_control/clock_stm32_ll_common.h

```diff
@@ -42,6 +42,33 @@
 #define z_plli2s_r(v) LL_RCC_PLLI2SR_DIV_ ## v
 #define plli2sr(v) z_plli2s_r(v)
 
+#define z_pllsai1_m(v) LL_RCC_PLLSAI1M_DIV_ ## v
+#define pllsai1m(v) z_pllsai1_m(v)
+
+#define z_pllsai1_p(v) LL_RCC_PLLSAI1P_DIV_ ## v
+#define pllsai1p(v) z_pllsai1_p(v)
+
+#define z_pllsai1_q(v) LL_RCC_PLLSAI1Q_DIV_ ## v
+#define pllsai1q(v) z_pllsai1_q(v)
+
+#define z_pllsai1_r(v) LL_RCC_PLLSAI1R_DIV_ ## v
+#define pllsai1r(v) z_pllsai1_r(v)
+
+#define z_pllsai2_m(v) LL_RCC_PLLSAI2M_DIV_ ## v
+#define pllsai2m(v) z_pllsai2_m(v)
```

Comment:
For the non-L4+, `pllsai2m` should be the same as `pllm`. You can use `RCC_PLLSAI1M_DIV_1_16_SUPPORT` to discriminate